test/randomized/test_transpiler_equivalence.py

```diff
@@ -13,18 +13,36 @@
 
 """Randomized tests of transpiler circuit equivalence."""
 
+import os
 from math import pi
 
+from distutils.util import strtobool
```

Comment:
`distutils` is deprecated as of Python 3.10, so we'll need to find another way to do this (it's probably simple enough that we can write our own function if necessary)

Reply:
Yeah, for this normally we just do something like:

```
string_value.upper() == "TRUE"
```

or if you want to get fancier

```
string_value.upper() == "TRUE" or string_value.upper() == "T" or string_value == "1"
```

but normally as long as you explicitly document what the accepted values are for the env variable to be used you're fine.

Reply:
Done in bdff868c3ac5ad0d85be74fe2ee95b54f72a2481.